libdnf.spec

```diff
@@ -345,17 +348,21 @@ Package management service with a DBus interface
 # ========== microdnf ==========
 
 %if %{with microdnf}
-%package -n microdnf
-Summary:        Package management service with a DBus interface
+%package -n microdnf5
```

Comment:
I would suggest leaving this as `microdnf` instead...

Reply:
That would break all installations (mainly containers) that use microdnf.
The current code is not ready yet.